test/api/index/index/001-index/expected-response.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
V tomto teste potom staci mat nejaku jednoduchu sablonu, myslim, ze je zbytocne zlozita, ... a ak sa bude kopirovat do dalsich testov, tak tam budeme mat vela zbytocnych suborov, a bude to menej prehladne.

Reply:
Jo pravda, jsem tam zkopíroval něco z `template-use` testů. Upravím ještě.